ArcGISRuntimeSDKQt_CppSamples/Analysis/DisplayUtilityAssociations/DisplayUtilityAssociations.cpp

```diff
@@ -0,0 +1,280 @@
+// [WriteFile Name=DisplayUtilityAssociations, Category=Analysis]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+#include "DisplayUtilityAssociations.h"
+
+#include "ArcGISFeatureTable.h"
+#include "AttributeListModel.h"
+#include "FeatureLayer.h"
+#include "GraphicsOverlay.h"
+#include "Map.h"
+#include "MapQuickView.h"
+#include "SimpleLineSymbol.h"
+#include "UniqueValueRenderer.h"
+#include "UtilityAssociation.h"
+#include "UtilityElement.h"
+#include "UtilityNetwork.h"
+#include "UtilityNetworkDefinition.h"
+#include "UtilityNetworkSource.h"
+#include "UtilityNetworkTypes.h"
+
+#include "SymbolImageProvider.h"
```

Comment:
this is your own type right? normally we'd put those up above the Runtime API includes